### DISCUSSION TITLE
Please transfer license key from chome to firefox

### DISCUSSION THREAD

**Discussion by tonydo2023** [aclap-dev/video-downloadhelper#1152] [License & Purchase requests]
### What's your issue?

Hey there! I'm very happy with this product, but i simply want to download videos from youtube and it seems that i can only do that with your extension for firefox, but my licence is for chrome. I'd like it to be transferred to firefox please.

### The 10 first digits (only 10) of your license number or email address

antiwet03@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
Sorry we can't swap licenses.
Chrome and Firefox premium are separate products with different prices and features.
